### ISSUE TITLE
not able to get The Path Untraveled trusts

### ISSUE THREAD

**Issue by helixhamin** [DerpyProjectGroup/topaz#656]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened
- [x] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I have specified what branch this happens on branch: base

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 
After completing The Path Untraveled RoV mission, you should be able to speak to Kupipi to get Cipher: Semih, which can be exchanged for Trust: Semih Lafihna, and to Halver to get a broken mission, or wait, maybe it is a trust. Cipher: Halver, which can be exchanged for trust HALVER!!!!

I want my trust halver, so I can go around breaking all the monster's missions too...

**Comment by RAIST5150** [comment]
More to it than just adding the cipher though... Even if someone added the cipher, he wouldn't do anything yet:

https://github.com/LandSandBoat/server/wiki/Trusts
(Linked from the What Works page of the wiki)

**Comment by helixhamin** [comment]
Yeah, I was going over trusts today, how I noted this, and note that right now it is all commented out. Anyways, this is mostly for reference xD
